pkg/subnet/upgrades/upgrade_file.go

```diff
@@ -14,19 +13,34 @@ import (
 	"github.com/ava-labs/avalanchego/utils/storage"
 )
 
+func WriteLockUpgradeFile(jsonBytes []byte, subnetName string, app *application.Avalanche) error {
```

Comment:
We manage all other file operations through the `application` package

Reply:
Also, it's very unclear to me what the contents of the lock file actually are. I suggest creating a lock file type (or reusing an existing type) and accepting that as an argument, then do any type conversions in this method, then finally call out to the app package to write the file.

Reply:
OK moving to `application` package__